cli/src/config.rs

```diff
@@ -0,0 +1,143 @@
+use color_eyre::Result;
+use hyper::{http::HeaderName, Method};
+use serde::Deserialize;
+use serde_with::{serde_as, DisplayFromStr};
+use std::{fs::File, io::BufReader, net::SocketAddr, path::PathBuf, time::Duration};
+use tower_http::cors::CorsLayer;
+
+pub type Timeout = u64;
+
+/// Server Configuration passed on initialization
+#[serde_as]
+#[derive(Clone, Debug, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub struct Config {
+    /// Server address <host:port>
+    pub address: SocketAddr,
+
+    /// Cors configuration
+    pub cors: Cors,
+
+    /// Max Time to Responce, in milliseconds
+    pub request_timeout: Timeout,
+
+    /// [Stub] File to save logs
+    pub log_file: Option<PathBuf>,
+
+    /// Tracing Level
+    #[serde(default = "tracing_default")]
+    #[serde_as(as = "DisplayFromStr")]
+    pub trace_level: tracing::Level,
```

Comment:
`trace_level` and `log_file` should be placed in a separate struct, so that in `config.yaml` the logger specific configuration will be inside `logger` section